src/library/dao/autodjcratesdao.cpp

```diff
@@ -15,6 +15,11 @@
 #include "mixer/playerinfo.h"
 #include "mixer/playermanager.h"
 
+#if !defined(VERBOSE_DEBUG_LOG)
```

Comment:
please don't invent a new log level.  "debug" is our verbose level, just use that.

Reply:
We need this to prevent log spam. DEBUG is currently used as INFO. Explained here #2782. Otherwise I would have to delete all those extra logs.

Reply:
Agree that we don't have a good story for Info vs Debug.  But this feels like a one-off change that will increase tech debt.  If you are going to do this, please create a bug and assign it to yourself to remove this special casing once we have better logging code.